### ISSUE TITLE
Question on reproducing results on SUNRGBD data

### ISSUE THREAD

**Issue by StarsMyDestination** [facebookresearch/votenet#66]
I followed the instructions in README to train VoteNet on SUNRGBD data. All settings keep the default, but batch_size changed to 128(8GPUs). After 180 epoches, I got  mAP results like this: 0.5466@IoU=0.25, 0.2811@IoU=0.5, which is much lower than that in paper(57 mAP@0.25 and 32 mAP@0.5).
If I use the pre-trained model, I can reproduce the results as paper noted, SO data is right.

Is batch_size matters so much? If so, how do I change the learning rate along with batch_size?
Any idea is welcome!
thanks!

**Comment by WhatAShot** [comment]
Have you solved this problem? How about using the default batch size? @StarsMyDestination
